internal/access/signup_test.go

```diff
@@ -96,12 +97,17 @@ func TestSignupEnabled(t *testing.T) {
 	pass := "password"
 
 	t.Run("SignupUser", func(t *testing.T) {
-		c, _ := setup(t)
+		c, db := setup(t)
 
 		enabled, err := SignupEnabled(c)
 		assert.NilError(t, err)
 		assert.Equal(t, enabled, true)
 
+		provider := models.Provider{Name: models.InternalInfraProviderName}
```

Comment:
This was relying on the provider being created in another test, added the explicit creation here